playground/DEPLOY.md

```diff
@@ -0,0 +1,95 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+## Manual App Install
+To install Playground applications into existed GCP account you'll need to follow several steps, in this document just simple set of commands will be provided that will guide you over install process.
```

Comment:
I think maybe we can improve this guide by changing the guide to the following steps (which I think are the requirements for manually installing the Playground app)

1) Building and pushing the backend docker image file to Google Artifact Registry (AR) 
2) Building and pushing frontend config files and docker image file to AR
3) Deploy services via terraform. 

Where each section describes the commands and variables and links to GCP guides if needed.